lib/ansible/modules/cloud/azure/azure_rm_managed_disk_facts.py

```diff
@@ -210,6 +210,19 @@ def list_items(self):
                 results.append(managed_disk_to_dict(item))
         return results
 
+    def list_items_by_resource_group(self):
```

Comment:
at least one test?

Reply:
@yungezz the test is there. and it just started failing, so this fix is fixing that test

Reply:
@zikalino why it didn't failed during time of module merging?